IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,32 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    button.active:not(button.active:first-child , .search button.active ){
+        margin-top:7px;
+    }
 }
-
+.side-control.search
+ {
+    button{
+        margin-top:2px;
+    }
+    button.active {
+        margin-top: 2px;
+    }
+ } 
 .input-wrapper-spacer {
     height: 0;
     margin-bottom: 1rem;
 }
-
+.mt-0{
```

Comment:
I don't think this is needed. See here:
https://tailwindcss.com/docs/margin

Reply:
The Tailwind utility class has low specificity and is overridden by our other CSS, which is why I used this custom class with !important.